src/Components/ProvisioningWizard/index.js

```diff
@@ -24,6 +27,11 @@ const ProvisioningWizard = ({ onClose, image, ...props }) => {
 
   const { isLoading, error: sourcesError, sources: availableSources } = useSourcesForImage(image, { refetch: stepIdReached <= 2 });
 
+  useEffectOnce(() => {
+    axios.defaults.headers.common['Provisioning-Wizard-Id'] = uuidv4();
+    return () => delete axios.defaults.headers.common['Provisioning-Wizard-Id'];
```

Comment:
Just for the record, I suggest a generic term that don’t have wizard in the name. Feel fee to just use "X-Correlation-Id" or something like "X-Custom-Id" or whatever is generic. This can be then used by any request in order to "tag" logs, not just by the wizard.